index.bs

```diff
@@ -1713,13 +1713,13 @@ with parameter |value| is:
 [=remote end definition=] and [=local end definition=]
 
 <pre class="cddl remote-cddl local-cddl">
-session.ProxyConfiguration = {
+session.ProxyConfiguration = (
```

Comment:
I think with this change we will be loosing the type information for the proxy field in session.CapabilityRequest. Instead, I think we can wrap the group choice into an extra `()` to make sure it is a group.